### ISSUE TITLE
View a work accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3967]
### Descriptive summary

In testing for the Hyrax 3.0 release, the following points are actions that need to be taken on the Hyrax view a work page in order to be more accessible: 

 **Elements must have sufficient color contrast:**
![Screen Shot 2019-09-11 at 5 47 07 PM](https://user-images.githubusercontent.com/9621070/64745299-333bad80-d4bc-11e9-9b2c-3179413be00f.png)

![Screen Shot 2019-09-11 at 5 47 55 PM](https://user-images.githubusercontent.com/9621070/64745316-523a3f80-d4bc-11e9-9f4b-a23c58108038.png)

![Screen Shot 2019-09-11 at 5 48 22 PM](https://user-images.githubusercontent.com/9621070/64745330-6120f200-d4bc-11e9-9945-b6db0d8be05f.png)

![Screen Shot 2019-09-11 at 5 48 50 PM](https://user-images.githubusercontent.com/9621070/64745353-6ed67780-d4bc-11e9-986d-9a8844b6b21a.png)

**Frames must have title attribute:**
![Screen Shot 2019-09-11 at 5 49 53 PM](https://user-images.githubusercontent.com/9621070/64745401-975e7180-d4bc-11e9-9485-919587973b81.png)

**Links must have discernible text:**
![Screen Shot 2019-09-11 at 5 51 04 PM](https://user-images.githubusercontent.com/9621070/64745444-bfe66b80-d4bc-11e9-9d71-6382031f6633.png)

**Zooming and scaling must not be disabled:**
![Screen Shot 2019-09-11 at 5 51 53 PM](https://user-images.githubusercontent.com/9621070/64745482-e3a9b180-d4bc-11e9-8039-c4bc76210ab0.png)

**ARIA role must be appropriate for the element:**
![Screen Shot 2019-09-11 at 5 52 45 PM](https://user-images.githubusercontent.com/9621070/64745519-089e2480-d4bd-11e9-8974-81e79692a049.png)

**Heading levels should only increase by one:**
![Screen Shot 2019-09-11 at 5 54 09 PM](https://user-images.githubusercontent.com/9621070/64745556-2cfa0100-d4bd-11e9-9820-1f98fdbe6d86.png)

**Ensures landmarks are unique:**
![Screen Shot 2019-09-11 at 5 56 20 PM](https://user-images.githubusercontent.com/9621070/64745641-7fd3b880-d4bd-11e9-992f-8591eb41ce3c.png)

**All page content must be contained by landmarks:**
![Screen Shot 2019-09-11 at 5 57 07 PM](https://user-images.githubusercontent.com/9621070/64745675-9712a600-d4bd-11e9-9270-9fa0ba9f2efc.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/concern/generic_works/th83kz52k?locale=en
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, unable to tab through buttons such as Feature, Analytics, Edit, File Manager, the links to the works, etc. 

In Chrome, tabbing works. 

![Screen Recording 2019-09-16 at 02 01 PM](https://user-images.githubusercontent.com/9621070/64993346-afead500-d88a-11e9-9654-180b5cc9886b.gif)
